### PR TITLE
Update the readme

### PR DESCRIPTION
**Pull request by anaelChardan**
**Description**

This follow this PR https://github.com/akeneo/transporteo/pull/58 adapting the Upgrade file with the Transporteo usage.

We do not support docker, pim installed locally...

**Definition Of Done**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: UPGRADE-2.0.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents:**
+
+- [Disclaimer](#disclaimer)
+- [Migrate your custom code](#migrate-your-custom-code)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+<!-- To update this content, execute `doctoc UPGRADE-1.7.md --title '**Table of Contents:**' --maxlevel 3` -->
+
+## Disclaimer
```
Comment (anaelChardan) [inline UPGRADE-2.0.md]: Interesting part from line 15 to 25

File: UPGRADE-2.0.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
+# UPGRADE FROM 1.7 to 1.8
```
Comment (nidup) [inline UPGRADE-2.0.md]: 1.7 to 2.0

File: README.md [hunk 3]
```diff
@@ -52,11 +53,14 @@ Moreover, if you migrate an _Enterprise Edition_, you'll be asked to provide a *
...
 
 ```bash
-  $ php MigrationTool.php akeneo-pim:migrate
+  $ php Transporteo.php akeneo-pim:migrate
 ```
 
 then, let you guide ;) 
 
+At the end of the tool's job, you will have your database setuped with your data but we do not migrate your custom code.
+You will have to migrate it following this [Upgrade file](./UPGRADE-2.0.md)
```
Comment (nidup) [inline README.md]: Add something like, we plan to automate this part in future release, stay tuned!

File: UPGRADE-2.0.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+- [Disclaimer](#disclaimer)
+- [Migrate your custom code](#migrate-your-custom-code)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+<!-- To update this content, execute `doctoc UPGRADE-1.7.md --title '**Table of Contents:**' --maxlevel 3` -->
+
+## Disclaimer
+
+> We assume that you ran Transporteo before, so you have currently two Akeneo PIM running, a 1.7 and a new 2.0
+> While the tool is not stable we do not recommend to use it to setup a Production PIM.
```
Comment (LaurentPetard) [inline UPGRADE-2.0.md]: I think that it should be at the beginning of the file, and highlighted

**Pull request merged**